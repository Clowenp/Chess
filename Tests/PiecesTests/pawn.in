p 4 1 4 2 1
p 4 1 4 3 0
p 4 1 4 0 0
p 7 4 8 4 3
