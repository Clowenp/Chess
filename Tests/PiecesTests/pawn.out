i am a black pawn at [4,1]
my possible moves are: 
< [4,0] >
< [3,0] >
< [5,0] >
the move to [4,2] is NOT possible
i am a white pawn at [4,1]
my possible moves are: 
< [4,2] >
< [5,2] >
< [3,2] >
the move to [4,2] is possible
i am a black pawn at [4,1]
my possible moves are: 
< [4,0] >
< [3,0] >
< [5,0] >
< >
the move to [4,3] is NOT possible
i am a white pawn at [4,1]
my possible moves are: 
< [4,2] >
< [5,2] >
< [3,2] >
< [4,3] >
the move to [4,3] is possible
i am a black pawn at [4,1]
my possible moves are: 
< [4,0] >
< [3,0] >
< [5,0] >
< >
the move to [4,0] is possible
i am a white pawn at [4,1]
my possible moves are: 
< [4,2] >
< [5,2] >
< [3,2] >
< [4,3] >
the move to [4,0] is NOT possible
i am a black pawn at [7,4]
my possible moves are: 
< [7,3] >
< [6,3] >
< >
the move to [8,4] is NOT possible
i am a white pawn at [7,4]
my possible moves are: 
< [7,5] >
< >
< [6,5] >
the move to [8,4] is NOT possible
